edge-middleware/bot-protection-datadome/README.md

```diff
@@ -5,7 +5,7 @@ description: DataDome can provide real-time bot protection and other security pr
 framework: Next.js
 useCase: Edge Middleware
 css: Tailwind
-deployUrl: https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fvercel-customer-feedback%2Fedge-middleware%2Ftree%2Fmain%2Fexamples%2Fbot-protection-datadome&env=NEXT_PUBLIC_DATADOME_CLIENT_KEY,DATADOME_SERVER_KEY&project-name=bot-protection-datadome&repository-name=bot-protection-datadome
+deployUrl: https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fvercel-customer-feedback%2Fedge-middleware%2Ftree%2Fmain%2Fexamples%2Fbot-protection-datadome&env=DATADOME_CLIENT_SIDE_KEY,DATADOME_SERVER_SIDE_KEY&project-name=bot-protection-datadome&repository-name=bot-protection-datadome
```

Comment:
Do we need the client side key? I see it's commented out in the example env file.

Reply:
Thanks, I removed the comment in the example file for consistency.